docs/section10/10-1.md

```diff
@@ -0,0 +1,42 @@
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:** None
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. The zone in the P-RMR includes humidification.
+
+**Function Call:** 
+
+1. match_data_element()
+2. get_zones_hvac_sys_humid_p()
+
+**Manual Check:** Check that the applicable baseline HVAC systems were modeled per the rules of G3.1-10 HVAC System baseline column which states that if the proposed design includes humidification then the baseline building design shall use adiabatic humidification (if the proposed humidification system does not comply with 90.1-2019 Section 6.5.2.4). For each applicable zone (i.e. zones with humidification in the proposed) check if the humidification system associated with the zone complies with 90.1-2019 Section 6.5.2.4. If it does NOT then check that the baseline system serving the zone was modeled with adiabatic humidification. 
+ 
+
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Call function to get list of applicable zones in P_RMR: `applicable_zones_hvac_sys_humid_dict_p = get_zones_hvac_sys_humid_p(P_RMR)`
+- For each zone in P_RMR: `zone_p in P_RMR...zones:`
+    - Check if the zone has humidification in the P_RMR, if so then applicability flag is true: `if applicable_zones_hvac_sys_humid_dict_p.has_key(zone_p.id): rule_applicability_flag = TRUE`
+    - If rule_applicability_flag = TRUE for zone_p: `if rule_applicability_flag == TRUE:`
+        - Get analogous zone ID in the B_RMR: `zone_b = match_data_element(B_RMR,Zones,zone_p.id)`
+        - For each terminal unit associated with the zone_b in the B_RMR: `for terminal_b in zone_b:`
+            - Get the served_by_heating_ventilation_air_conditioning_systems for each terminal: `heating_ventilation_air_conditioning_systems_b = terminal_b.served_by_heating_ventilation_air_conditioning_systems`
+            - Add to list of heating_ventilation_air_conditioning_systems_list_b as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_b = heating_ventilation_air_conditioning_systems_list_b.append(heating_ventilation_air_conditioning_systems_b)`                
+- Convert the list of heating_ventilation_air_conditioning_systems_list_b associated with the applicable zones to a set and the back to a list to eliminate duplicates after looping through all zones: `heating_ventilation_air_conditioning_systems_list_b = list(set(heating_ventilation_air_conditioning_systems_list_b))`
+- For each hvac_b in heating_ventilation_air_conditioning_systems_list_b: `For in heating_ventilation_air_conditioning_systems_list_b:` 
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, outcome is undetermined, manual check needs to be performed: `UNDETERMINED`
+    
```

Comment:
@claperle and @mkarpman we could check the humidification type for the baseline HVAC system and identify the ones which are adiabatic and non adiabatic. we could then identify the corresponding proposed HVAC systems and identify the ones which need to comply with the prescriptive requirements and the ones that dont. That might help provide more information. We're already getting all the prerequisite data points here.

Reply:
@mcneillj we should probably add a category up to which identifies a non-mandatory rule. I didn't realize this one was a non-mandatory rule and instead assumed the undetermined outcome was due to other limitations (which is also the case in this one)

Reply:
I'll wait for a discussion on this with James and the group.

Reply:
as @mkarpman summarized in the email, #248  could be merged with #250

Reply:
All comments implemented